nservicebus/hosting/cloud-services-host/index.md

```diff
@@ -21,145 +21,45 @@ The Azure Platform and NServiceBus make a perfect fit. On the one hand the Azure
 
 If real scale is required (eg in tens, hundreds or even thousands of machines hosting each endpoint) than cloud services is the required deployment model.
 
-NOTE: If this level of scale is not required, there are [other hosting options available](hosting-options.md).
-
-
 ## Cloud Services - Worker Roles
 
 Reference the assembly that contains the Azure role entry point integration. The recommended way of doing this is by adding a NuGet package reference to the `NServiceBus.Hosting.Azure` package to the project.
 
-NOTE: If self hosting, done later in this article for Web Roles, everything can be configured using the configuration API and the extension methods found in the `NServiceBus.Azure` package, no need to reference the hosting package in that case.
+NOTE: If self hosting, done later in this article for Web Roles, everything can be configured using the configuration API and the extension methods found in the nservicebus azure related packages, no need to reference the hosting package in that case.
 
 To integrate the NServiceBus generic host into the worker role entry point, create a new instance of `NServiceBusRoleEntrypoint` and call it's `Start` and `Stop` methods in the appropriate `RoleEntryPoint` override.
 
 snippet:HostingInWorkerRole
 
 Next to starting the role entry point, define the endpoint behavior. The role has been named `AsA_Worker`. Specify the transport to use, using the `UseTransport<T>`, as well the persistence using the `UsePersistence<T>` configuration methods.
 
-snippet:ConfigureEndpoint
+snippet:AzureServiceBusTransportWithAzureHost
 
 This will integrate and configure the default infrastructure:
 
  * Configuration setting will be read from the `app.config` file, merged with the settings from the service configuration file.
- * Logs will be sent to the `TraceLogger`, the latter should have been implemented with Azure diagnostic monitor trace listener by the visual studio tooling.
+ * Logs will be sent to the `Trace` infrastructure, which should have been configured with Azure diagnostic monitor trace listener by the visual studio tooling.
```

Comment:
There is a link 3 lines below towards our logging page, which refers to the logging pages of MS (there are multiples now depending on version)